### PR TITLE
Add support for card labels on Trello and Slack integration

### PR DESCRIPTION
**Pull request by pedro-teixeira**

### CODE REVIEW

File: tests/models/trelloTest.js [hunk 1]
```diff
@@ -1076,4 +1076,88 @@ exports.testModelTrello = {
...
+        function ( error, response ) {
+          test.strictEqual( error, true );
+          test.strictEqual( response.test, true );
+        }
+      );
+
+      test.done();
+    },
+  },
 };
```
Comment (stefanjudis) [inline tests/models/trelloTest.js]: Ha ha - just once the last time.

Missing new line. :bowtie:

Comment (pedro-teixeira) [inline tests/models/trelloTest.js]: There is no new line at all files, jshint is not complaining too.

**Comment by pedro-teixeira** [comment]
New lines added

File: tests/models/trelloTest.js (hunk 1 above)
Comment (stefanjudis) [inline tests/models/trelloTest.js]: Damn it. Okay. :+1: :)

Pedro Teixeira <notifications@github.com> schrieb am Mi., 18. Feb. 2015
22:37:

**Comment by sgabler** [comment]
:+1:

**Comment by pedro-teixeira** [comment]
Please review it again, I've added Slack integration.

**Comment by sgabler** [comment]
:+1:

**Pull request merged**